client/extensions/zoninator/components/settings/zone/index.jsx

```diff
@@ -1,5 +1,175 @@
-const Zone = () => {
-	return null;
-};
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import page from 'page';
+import { localize } from 'i18n-calypso';
+import { flowRight } from 'lodash';
+import { FieldArray, FormSection, reduxForm } from 'redux-form';
 
-export default Zone;
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import FormButton from 'components/forms/form-button';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import FormTextarea from 'components/forms/form-textarea';
+import HeaderCake from 'components/header-cake';
+import RecentPostsDropdown from './recent-posts-dropdown';
+// import ReduxFormTextarea from 'components/redux-forms/redux-form-textarea';
+import ReduxFormTextInput from 'components/redux-forms/redux-form-text-input';
+import SearchAutocomplete from './../../search-autocomplete';
+import SectionHeader from 'components/section-header';
+import SortableList from 'components/forms/sortable-list';
+
+const form = 'editZone';
+
+class Zone extends Component {
+
+	addPost = ( { push } ) => {
+		return ( post ) => push( post );
+	}
+
+	removePost = ( { remove } ) => {
+		return ( idx ) => remove( idx );
+	}
+
+	changePostOrder = ( { move } ) => {
+		return ( order ) => {
+			let from = 0;
+
+			while ( order[ from ] < order.length - 1 && order[ from ] < order[ from + 1 ] ) {
+				from++;
+			}
+
+			let to = from + 1;
+
+			while ( to < order.length && order[ to ] > order[ to - 1 ] ) {
+				to++;
+			}
+
+			move( from, to );
+		};
+	}
+
+	handleGoBack = () => {
+		page( `/extensions/zoninator/${ this.props.siteSlug }` );
+	}
+
+	handleMouseDown = ( event ) => {
+		event.stopPropagation();
+		event.preventDefault();
+	}
+
+	renderPosts = ( { fields } ) => {
+		const { translate } = this.props;
+		const posts = fields.getAll() || [];
+		const showPosts = posts.length > 0;
+
+		const renderPost = ( { title, slug }, idx ) => (
+			<SectionHeader key={ slug } label={ title } className="zone__list-item">
+				<Button compact>{ translate( 'View' ) }</Button>
+				<Button compact>{ translate( 'Edit' ) }</Button>
+				<Button
+					compact
+					scary
+					onMouseDown={ this.handleMouseDown }
+					onClick={ this.removePost( fields ).bind( this, idx ) }>
+					{ translate( 'Remove' ) }
+				</Button>
+			</SectionHeader>
+		);
+
+		return (
+			<div>
+				<FormFieldset>
+					<FormLabel>
+						{ translate( 'Add content' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **28** times:
`translate( 'Add Content' )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/zoninator-edit-zone)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).